cli/snapshot.go

```diff
@@ -32,28 +32,37 @@ import (
 )
 
 const (
-	errMsgDataPartF = "Device-local data is stored on the partition " +
-		"being snapshotted: %s. The recommended approach is to have " +
-		"a separate data-partition mounted on \"/data\" and add a " +
+	// Error messages //
+	errMsgDataPartFmt = "Device-local data is stored on the rootfs " +
+		"partition: %s. The recommended approach is to have  a " +
+		"separate data-partition mounted on \"/data\" and add a " +
 		"symbolic link (%s -> /data). https://docs.mender.io/devices/" +
 		"general-system-requirements#partition-layout"
+	errMsgThawFmt = "CRITICAL: Unable to unfreeze filesystem, try " +
+		"running `fsfreeze -u %s` or press `SYSRQ+j`, immediately!"
 
 	// Watchdog constants //
-
-	// WDTExpired is set by the listening routine if the timer expires.
-	WDTExpired int32 = -1
-	// WDTReset is swapped in on read by the listening routine.
-	WDTReset int32 = 0
-	// WDTSet is set by the monitored process.
-	WDTSet int32 = 1
-	// WDTIntervalSec Sets the polling interval of the listening routine.
+	// wdtExpired is set by the listening routine if the timer expires.
+	wdtExpired int32 = -1
+	// wdtReset is swapped in on read by the listening routine.
+	wdtReset int32 = 0
+	// wdtSet is set by the monitored process.
+	wdtSet int32 = 1
+	// wdtIntervalSec Sets the polling interval of the listening routine.
 	// NOTE: this should be sufficient enough for the user to for example
 	//       type their password to ssh etc.
-	WDTIntervalSec = 30
+	wdtIntervalSec = 30
+
+	// bufferSize for the Copy function
+	bufferSize = 32 * 1024
 )
 
 var (
 	errWatchDogExpired = fmt.Errorf("watchdog timer expired")
+
+	// pbCondition is used to determine when the progress bar can start
+	// printing. This value is nil if the snapshot is not dumped to a pipe.
+	pbCondition utils.StartCondition
```

Comment:
Could we find a way to pass this in the arguments instead? Or in a struct? I don't think this should be global.